### ISSUE TITLE
Alias usage in resources

### ISSUE THREAD

**Issue by isukces** [AmmyUI/AmmyUI#17]
Alias used inside resource dictionary is translated without Key property.
Example:
1. Alias definition:
```
   alias MyDoubleValidation(name, min, max) {
                DoubleValidation {
                          CanBeNull: "False"
                          MinValue: $min
                          MaxValue: $max
                          ValueName: $name
                }
}
```
2. Alias used in dictionary:
```
Resources:ResourceDictionary {
	@AnyAlias("A", "1", "2") Key="MyKey" {}
}
```
3. XAML translation 
```
<ns0:DoubleValidation xmlns:ns0="clr-namespace:isukces.Wpf.Coolstuff;assembly=isukces.Wpf.Coolstuff" CanBeNull="False" MinValue="1" MaxValue="2" ValueName="A"/>
```
Without Key="MyKey"

**Comment by ionoy** [comment]
Fixed in 1.1.16

Thanks for feedback!

**Comment by isukces** [comment]
Thank you. It works.
